pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>groupId</groupId>
     <artifactId>kaitai_struct_visualizer_java</artifactId>
-    <version>0.5-SNAPSHOT</version>
+    <version>0.8-SNAPSHOT</version>
```

Comment:
0.8 was already released — if you want to use 0.8 APIs (which makes sense), then we'd rather want to use a `0.8` release version here, not 0.8-SNAPSHOT?

Reply:
I upgrade version of runtime dependency to released `0.8`. kaitai compiler dependency itself has no released `0.8` version and that line is just `kaitai_struct_gui` version, that, I think, has no released version itself, so no need to remove `-SNAPSHOT` here

Reply:
This is actually a very good point — I thought that it was done, but it turns out that it was never the case. Let me see if I'll be able to publish it properly.

Reply:
Ok, it takes some more time than I've anticipated. Ultimately, I guess, at least we'll have some solid instructions on how to publish that...

Reply:
Ok, 0.8 was published finally, so I guess this is resolved. Please check if it works for you?

Reply:
Mmm, my maven can not find `kaitai-struct-compiler_2.11-0.8` version. Actually, even with `kaitai-struct-runtime-0.8` visualizer not worked. It compiles fine, but when I try to load KSY file, generated class can not be compiled, because of strange errors:
```
/io/kaitai/struct/visualized/Ddc.java:5: error: cannot find symbol
import io.kaitai.struct.ByteBufferKaitaiStream;
                       ^
  symbol:   class ByteBufferKaitaiStream
  location: package io.kaitai.struct
...

100 errors
Exception in thread "main" java.lang.RuntimeException: Unknown error during compilation.
	at org.mdkt.compiler.InMemoryJavaCompiler.compileAll(InMemoryJavaCompiler.java:61)
	at org.mdkt.compiler.InMemoryJavaCompiler.compile(InMemoryJavaCompiler.java:73)
	at io.kaitai.struct.visualizer.VisualizerPanel.compileAndLoadJava(VisualizerPanel.java:115)
	at io.kaitai.struct.visualizer.VisualizerPanel.parseFileWithKSY(VisualizerPanel.java:120)
	at io.kaitai.struct.visualizer.VisualizerPanel.loadAll(VisualizerPanel.java:60)
	at io.kaitai.struct.visualizer.MainWindow.main(MainWindow.java:24)
```
With `kaitai-struct-runtime-0.8-SNAPSHOT` all works.